secv_guis/base_widgets.py

```diff
@@ -48,19 +48,22 @@ def __init__(self, label, parent=None,
         self.file_watcher.directoryChanged.connect(
             lambda: self.update_path(self.dirpath))
 
-    def update_path(self, dirname,file_filter=None):
+    def update_path(self, dirname,selected_image=None):
         """
         :param str dirname: The new directory path to be listed.
+        :param selected_images: The name of an image with which the list can be filtered
         """
-        if file_filter is None:
+        if selected_image is None:
             file_names = [f for f in os.listdir(dirname)
-                      if f.lower().endswith(tuple(self.extensions))]
+                          if f.lower().endswith(tuple(self.extensions))]
```

Comment:
Linting: I believe the start of this line has some sort of tab? if that is the case please do not use tabs, that only creates headaches. You can probably configure your text editor to replace tab with multiple spaces, that's the usual convention